### DISCUSSION TITLE
Reporting detailed information about the deployment

### DISCUSSION THREAD

**Discussion by tovbinm** [fluxcd/flux2#2643] [Q&A] [answered]
Hi 👋🏻 folks,

We would like to use Flux notifications to report information about the deployments to [Faros](https://github.com/faros-ai/faros-community-edition). 

Our Flux deployment uses Source Controller to pull information from a GitHub repo and initiate a deployment with Kustomize Controller. We subscribed to the notifications produced by the Source Controller and Kustomize Controller but they do not contain enough information:
```
{
  "involvedObject": {
    "kind":"GitRepository",
    "namespace":"flux-system",
    "name":"flux-system"
    // ...
  },
  "timestamp":"2021-04-15T01:04:05Z",
  "message":"Fetched revision: <branch>/<commit>",
  "reportingController":"source-controller"
  // ...
}
```

Is it possible to extend these notifications event with more information about the deployment, such as GitHub org & repo names, Docker artifact tags & labels being deployed, service name, environment etc?

**Comment by kingdonb** [comment, marked as answer]
There is `Alert.spec.summary` which is a simple string, at this time there's no templating here. There has been some discussion about extending this.

```
	// Short description of the impact and affected cluster.
	// +optional
	Summary string `json:"summary,omitempty"`
```

This is not much help, unfortunately. I think this is a topic of interest. If you could elaborate on your use case, you've provided some good examples of datapoints you'd like to use, it would be good to get a clearer picture of how they will be used to make sure we are addressing the issue with any improvements that might be planned!

One idea is to use this type of information together with the new [Dispatch Trigger for GitHub Workflows](https://fluxcd.io/docs/components/notification/provider/#setting-up-the-github-dispatch-provider)
